Rawl_B502/clann_faílbe_id_est_clann_donngaile.trig

```diff
@@ -386,8 +386,7 @@
     a foaf:Person;
     irishRel:genName "Bressail";
     irishRel:nomName "Bressal";
-    rel:childOf <#Óengusa-49ac7069>;
-    owl:sameAs <http://example.com/LL/eoganachta_casil.trig#Bresail>.
+    rel:childOf <#Óengusa-49ac7069>.
```

Comment:
This is like the above.  All the other people in this line are from the Eoganachta except one.